src/api/app/controllers/webui/package_controller.rb

```diff
@@ -246,7 +246,7 @@ def submit_request_dialog
   end
 
   def submit_request
-    required_parameters :project, :package
+    required_parameters :project, :package, :targetproject
```

Comment:
``target_project_name = params[:targetproject].strip`` will raise a nil pointer exception so I assumed the ``:targetproject`` is mandatory! However, I'm fine with dropping this line for this PR and fix it in another PR later ...